packages/playwright-webkit/index.js

```diff
@@ -13,12 +13,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+const path = require('path');
 const {Playwright} = require('playwright-core/lib/server/playwright.js');
 
-module.exports = new Playwright({
-  downloadPath: __dirname,
+const playwright = new Playwright({
   browsers: ['webkit'],
-  respectEnvironmentVariables: true,
```

Comment:
Can we pass the `.downloaded-browser.json` path in here to avoid having logic in the `playwright-<browser>` packages?

Reply:
I'd rather not; we try to isolate playwright-core from knowing about downloads.